docs/ref/contrib/admin/index.txt

```diff
@@ -34,11 +34,17 @@ If you're not using the default project template, here are the requirements:
 
 #. Configure a :class:`~django.template.backends.django.DjangoTemplates`
    backend in your :setting:`TEMPLATES` setting with
-   ``django.contrib.auth.context_processors.auth`` and
+   ``django.contrib.auth.context_processors.auth``,
+   ``django.contrib.messages.context_processors.request``, and
```

Comment:
Maybe:
```
``django.contrib.messages.context_processors.request` (required for the navigation sidebar)
```

Reply:
Not sure... I think it's possibly overkill for a new user just trying to make sure the admin works. (It's an implementation detail as to WHY it's needed...) 

"Just put these in..." 

🤷

Reply:
Agreed.

Reply:
I'll leave this given the adjustment to the versionchanged.